src/pyqasm/modules/base.py

```diff
@@ -259,11 +260,13 @@ def remove_includes(self, in_place=True) -> Optional["QasmModule"]:
 
         return curr_module
 
-    def depth(self):
+    def depth(self, decompose_gates=True):
```

Comment:
```suggestion
    def depth(self, decompose_native_gates=True):
```

Reply:
Could you rename this as well? The parameter is user facing and it is better to specify the same name as the one used in the `QasmModule`